_config.yml

```diff
@@ -365,6 +365,14 @@ apps:
     ds_type:
       - raw
 
+  - gh: "poldracklab/fitlins"
+    dh: "poldracklab/fitlins"
+    branch: "dev"
+    ds_type:
+      - derivative
```

Comment:
It needs both raw and derivative. Does that make a difference here?

Reply:
hum...
This is mostly needed to create the subtables where apps are sorted by input type.

See for example
https://bids-apps.neuroimaging.io/apps/#raw-anat

Most of those tables expect that only one type of datasets is required
I may need to find a way to have tables for the apps that need raw AND derivative.

Reply:
Nah, it's fine. I expect most second order apps also require some info from the raw dataset.